src/background/service/userWallet.ts

```diff
@@ -797,10 +828,10 @@ class UserWallet {
 
```

Comment:
why we convert the hex string to buffer, and then convert it back to hex string??

```js
const realSignature = await signWithKey(
      Buffer.from(signableMessage, 'hex').toString('hex'),
...
```